manager/assets/modext/widgets/security/modx.panel.user.js

```diff
@@ -437,8 +476,49 @@ Ext.extend(MODx.panel.User,MODx.FormPanel,{
                     ,anchor: '100%'
                 }]
             }]
-        });
-        return [{
+        }]);
+
+        var itemsLeft = [{
+            id: 'modx-user-id'
+            ,name: 'id'
+            ,xtype: 'hidden'
+            ,value: config.user
+        },{
+            id: 'modx-user-username'
+            ,name: 'username'
+            ,fieldLabel: _('username')
+            ,description: _('user_username_desc')
+            ,xtype: 'textfield'
+            ,anchor: '100%'
+            ,listeners: {
+                'keyup': {scope:this,fn:function(f,e) {
+                    Ext.getCmp('modx-user-header').getEl().update(_('user')+': '+f.getValue());
```

Comment:
As with my other comment, this also needs to protect against XSS in `f.getValue()`

Reply:
Done